frontend/src/components/VaultList.vue

```diff
@@ -18,17 +18,17 @@
     <input id="vaultSearch" v-model="query" :placeholder="t('vaultList.search.placeholder')" type="text" class="focus:ring-primary focus:border-primary block w-full shadow-xs text-sm border-gray-300 rounded-md disabled:bg-gray-200"/>
 
     <Listbox v-model="selectedFilter" as="div">
-      <div class="relative w-44">
-        <ListboxButton class="relative w-full rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 text-left shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary text-sm">
-          <span class="block truncate">{{ filterOptions[selectedFilter] }}</span>
+      <div class="relative w-auto whitespace-nowrap">
+        <ListboxButton class="min-w-[250px] relative w-full rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 text-left shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary text-sm">
```

Comment:
We should use Tailwind’s default spacing scale here and not fixed pixel values:

```suggestion
        <ListboxButton class="min-w-60 relative w-full rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 text-left shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary text-sm">
```